line_item_manager/conf.d/line_item_template.yml

```diff
@@ -49,6 +49,23 @@ targeting:
       {% for placement in li.gam.placements|sort(attribute='id') %}
       - {{ placement.id }}
       {% endfor %}
+  {% if li.gam.geographies %}
+  geoTargeting:
+    {% if li.gam.geographies.include|length != 0 %}
+    includedLocations:
+      {% for geo in li.gam.geographies.include %}
+      - xsi_type: "Location"
+        id: {{ geo.id }}
+      {% endfor %}
+    {% endif %}
+    {% if li.gam.geographies.exclude|length != 0 %}
+    excludedLocations:
```

Comment:
I am not seeing this key supported here:
https://developers.google.com/ad-manager/api/reference/v202208/LineItemService.GeoTargeting

Reply:
this is true for `includedLocations` , but `excludedLocations` is definitely defined in https://developers.google.com/ad-manager/api/reference/v202208/LineItemService.GeoTargeting